cypress/e2e/project-list.cy.ts

```diff
@@ -63,5 +65,22 @@ describe("Project List", () => {
             .should("have.attr", "href", "/dashboard/issues");
         });
     });
+
+    it("renders error component", () => {
+      cy.intercept(projectListEndpoint, { statusCode: 500 }).as(
+        "getServerFailure",
+      );
+      cy.visit("http://localhost:3000/dashboard");
+
+      // cy.get('[data-cy="error-component"]', { timeout: 10000 });
+
+      // react-query has a default retry value of 3 times
+      cy.wait("@getServerFailure")
+        .wait("@getServerFailure")
+        .wait("@getServerFailure")
+        .wait("@getServerFailure");
```

Comment:
You could set a long enough timeout in the `cy.get` so that the tests waits until the error appears. That way you could get around these waits